### ISSUE TITLE
Can we tell futurize to ignore/skip a specific line?

### ISSUE THREAD

**Issue by yongjik** [PythonCharmers/python-future#491]
Hi, I need a way to mark a small portion of code as "2.7 only" and tell futurize to skip it. (That part of the code will be there until we fully migrate to Python 3.) Basically something like this:

```
import sys

def f(x):
    if sys.version_info[0] == 2:
        if type(x) == unicode:  # futurize: please don't change this line.
            call_some_py2_specific_logic()
    the_rest_of_logic()
```

I want futurize to not change "unicode" to "str" on that line.  Is there a relatively painless way to achieve this?

...Apologies if this is not the right place. I asked the same question on stackoverflow ( https://stackoverflow.com/questions/57498813/ ) but didn't get anything.

**Comment by Amir-Hadadi** [comment]
Ever found an answer?

**Comment by yongjik** [comment]
> Ever found an answer?

No, I ended up writing a small Python script that recognizes my own "don't change" mark, runs futurize, and then use difflib to paste the original code back on selected parts.  (I actually ended up using modernize, but that's orthogonal..)

It's arguably silly, but it gets the job done. :)

**Comment by adeadman** [comment]
This does limit the utility of `futurize` as a linter in a CI/CD pipeline (to prevent regressions in codebases that need to maintain Py2/3 compatibility).

However, to ignore *this specific fixer* you can also run something like `futurize -2 -x libfuturize.fixes.fix_unicode_keep_u` which prevents the unicode fixer from running - if you're happy that you've moved all of them over, that might be ok.
